files/en-us/web/http/cors/index.html

```diff
@@ -391,11 +391,11 @@ <h3 id="Access-Control-Allow-Origin">Access-Control-Allow-Origin</h3>
 <pre class="brush: none">Access-Control-Allow-Origin: https://mozilla.org
 Vary: Origin</pre>
 
-<p>If the server specifies a single origin (that may dynamically change based on the requesting origin as part of a white-list) rather than the "<code>*</code>" wildcard, then the server should also include <code>Origin</code> in the {{HTTPHeader("Vary")}} response header — to indicate to clients that server responses will differ based on the value of the {{HTTPHeader("Origin")}} request header.</p>
+<p>If the server specifies a single origin (that may dynamically change based on the requesting origin as part of a allowlist) rather than the "<code>*</code>" wildcard, then the server should also include <code>Origin</code> in the {{HTTPHeader("Vary")}} response header — to indicate to clients that server responses will differ based on the value of the {{HTTPHeader("Origin")}} request header.</p>
 
 <h3 id="Access-Control-Expose-Headers">Access-Control-Expose-Headers</h3>
 
-<p>The {{HTTPHeader("Access-Control-Expose-Headers")}} header lets a server whitelist headers that Javascript (such as {{domxref("XMLHttpRequest.getResponseHeader()","getResponseHeader()")}}) in browsers are allowed to access.</p>
+<p>The {{HTTPHeader("Access-Control-Expose-Headers")}} header lets a server allowlist headers that Javascript (such as {{domxref("XMLHttpRequest.getResponseHeader()","getResponseHeader()")}}) in browsers are allowed to access.</p>
```

Comment:
```suggestion
<p>The {{HTTPHeader("Access-Control-Expose-Headers")}} header adds the headers to the allowlist that Javascript (such as {{domxref("XMLHttpRequest.getResponseHeader()","getResponseHeader()")}}) in browsers is allowed to access.</p>
```

Reply:
I'm trying to avoid "lets a server allowlist ..." which just reads strangely. If you think that sounds ok please commit it.

Reply:
What's your opinion on this one:
```suggestion
<p>The {{HTTPHeader("Access-Control-Expose-Headers")}} header adds the specified headers to the allowlist that Javascript (such as {{domxref("XMLHttpRequest.getResponseHeader()","getResponseHeader()")}}) in browsers is allowed to access.</p>
```
(Added the *specified* there).
Also, I'll change *Javascript* to *JavaScript* in a separate commit.

Reply:
yep that looks good :)